app/controllers/events_controller.rb

```diff
@@ -25,15 +34,15 @@ def event_params
     params.require(:event).permit(:title, :description, :start_date, :end_date)
   end
 
-  def update_events_list(format)
-    format.turbo_stream do
-      render turbo_stream: [
-        turbo_stream.prepend(:events_list,
-                             partial: 'partials/events/card',
-                             locals: { event: @event }),
-        turbo_stream.update(:new_event_form, '')
-      ]
-    end
-    format.html { redirect_to events_path, notice: 'Event was successfully created.' }
+  def event
+    Event.find(params[:id])
+  end
+
+  def event_to_unattend
+    EventAttendee.find_by(
+      'attendee_id=? AND attended_event_id=?',
+      current_user.id,
+      @event.id
+    )
```

Comment:
this method is not being used. It can be removed
```suggestion
```

Reply:
right! done!